### PR TITLE
updates all aside text to sentence case

### PR DESCRIPTION
**Pull request by sarah11918**
#### Description (required)

This changes all of our aside (e.g. "tip", "note" etc.) custom titles to sentence case.

The Astro Docs site automatically converts all text written as a `:::note[custom title]` to uppercase, as a design choice.

Starlight, however, displays text exactly as written. Moreover, the design choice for Starlight is to NOT show this text as all upper case. So, existing asides had to be checked to make sure all custom title text was written in sentence case.

We used @vasfvitor 's find and replace algorithm to convert every custom title to sentence case, then manually inspected for proper nouns (e.g. Astro, StackBlitz, RSS) and manually added capitals.

image

### CODE REVIEW

File: src/content/docs/en/guides/imports.mdx [hunk 1]
```diff
@@ -51,7 +51,7 @@ Astro includes built-in support for [TypeScript](https://www.typescriptlang.org/
 
 **Astro doesn't perform any type checking itself.** Type checking should be taken care of outside of Astro, either by your IDE or through a separate script. For type checking Astro files, the [`astro check` command](/en/reference/cli-reference/#astro-check) is provided.
 
-:::note[TypeScript and file extensions]
+:::note[Typescript and file extensions]
```
Comment (sarah11918) [inline src/content/docs/en/guides/imports.mdx:54]:
```suggestion
:::note[TypeScript and file extensions]
```

File: src/content/docs/en/guides/testing.mdx [hunk 2]
```diff
@@ -100,7 +100,7 @@ export default defineConfig({
     });
     ```
 
-    :::tip[set a baseUrl]
+    :::tip[Set a baseurl]
```
Comment (vasfvitor) [inline src/content/docs/en/guides/testing.mdx]:
```suggestion
    :::tip[Set a baseUrl]
```
`baseUrl` is the key name from Cypress

Comment (sarah11918) [inline src/content/docs/en/guides/testing.mdx:103]:
```suggestion
    :::tip[Set a `baseUrl`]
```

File: src/content/docs/en/guides/testing.mdx [hunk 3]
```diff
@@ -211,7 +211,7 @@ Alternatively, you can install Playwright within your Astro project using the pa
     });
     ```
 
-    :::tip[set a baseurl]
+    :::tip[Set a baseurl]
```
Comment (sarah11918) [inline src/content/docs/en/guides/testing.mdx:214]:
```suggestion
    :::tip[Set a `baseUrl`]
```

File: src/content/docs/en/reference/cli-reference.mdx [hunk 4]
```diff
@@ -309,7 +309,7 @@ The `clear` command resets the telemetry data:
 astro telemetry clear
 ```
 
-:::tip[Want to disable telemetry in CI environments?]
+:::tip[Want to disable telemetry in ci environments?]
```
Comment (sarah11918) [inline src/content/docs/en/reference/cli-reference.mdx:312]:
```suggestion
:::tip[Want to disable telemetry in CI environments?]
```

File: src/content/docs/en/guides/testing.mdx (hunk 3 above)
Comment (vasfvitor) [inline src/content/docs/en/guides/testing.mdx]:
```suggestion
    :::tip[Set a baseURL]
```
now it's playwright `baseURL` key

**Review by vasfvitor — APPROVED** [review]

**Comment by vasfvitor** [comment]
LGTM!

File: src/content/docs/en/guides/images.mdx [hunk 5]
```diff
@@ -62,7 +62,7 @@ Images in the `public/` folder, as well as remote images not specifically config
 
 `<Image />` can transform a local or authorized remote image's dimensions, file type, and quality for control over your displayed image. The resulting `<img>` tag includes `alt`, `loading`, and `decoding` attributes and infers image dimensions to avoid **Cumulative Layout Shift (CLS)**.
 
-:::tip[What is Cumulative Layout Shift?]
+:::tip[What is cumulative layout shift?]
```
Comment (sarah11918) [inline src/content/docs/en/guides/images.mdx:65]:
```suggestion
:::tip[What is Cumulative Layout Shift?]
```

**Pull request merged**